tips.json

```diff
@@ -52,6 +52,18 @@
       "tip": "git remote set-url origin <URL>"
   },
   {
+    "title": "Get list of all remote references",
+    "tip": "git remote || git remote show"
```

Comment:
^ makes sense, BTW the pre-commit hook is not working for her, are we missing something ?